test/common/router/router_test.cc

```diff
@@ -4705,6 +4705,51 @@ TEST_P(RouterShadowingTest, NoShadowForConnect) {
   router_->onDestroy();
 }
 
+TEST_P(RouterShadowingTest, LahDeeDah) {
```

Comment:
Please use a better test name :)
and consider adding a one-line comment explaining this test.

Reply:
Whoopsie, sorry. Done.